icechunk-python/examples/dask_write.py

```diff
@@ -188,13 +188,15 @@ def update(args: argparse.Namespace) -> None:
     client = Client(n_workers=args.workers, threads_per_worker=1)
 
     map_result = client.map(execute_write_task, tasks)
-    worker_stores = client.gather(map_result)
+    worker_changes = client.gather(map_result)
 
     print("Starting distributed commit")
     # we can use the current store as the commit coordinator, because it doesn't have any pending changes,
     # all changes come from the tasks, Icechunk doesn't care about where the changes come from, the only
     # important thing is to not count changes twice
-    commit_res = store.distributed_commit("distributed commit", [ws.change_set_bytes() for ws in worker_stores])
+    for changes in worker_changes:
+        store.merge(changes)
```

Comment:
Thoughts on `store.merge(stores)` instead? Is there a benefit to merging the changeset bytes?

Reply:
I am ok with it, I was just thinking of use cases where the full stores aren't serialized but it's kind of a non issue

Reply:
I agree with it being a non-issue